src/app/api/public/webhooks/stripe/route.ts

```
@@ -0,0 +1,239 @@
+
+type StripeInvoice = Stripe.Invoice & {
+  subscription: string;
+};
+
+// Helper function to create or update a plan
+async function createOrUpdatePlan(product: Stripe.Product) {
+  const features = product.metadata?.features
+    ? JSON.parse(product.metadata.features)
+    : [];
+
+  delete product.metadata?.features;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.



(lint/performance/noDelete)